### PR TITLE
French translation is up to date. 

### PR DESCRIPTION
**Pull request by PascalGrosse**
Also fixed a lot of small typos and translation/grammatical errors.

### CODE REVIEW

File: translations/base-fr.yaml [hunk 1]
```diff
@@ -391,25 +392,25 @@ ingame:
                 Connectez l'extracteur avec un <strong>convoyeur</strong> vers votre centre !<br><br>Astuce: <strong>Cliquez et faites glisser</strong> le convoyeur avec votre souris !
 
             1_3_expand: >-
-                Ceci n'est <strong>PAS</strong> un jeu incrémental et inactif ! Construisez plus d'extracteurs et de convoyeurs pour atteindre plus vite votre votre but.<br><br>Astuce: Gardez <strong>SHIFT</strong> enfoncé pour placer plusieurs extracteurs, et utilisez <strong>R</strong> pour les faire pivoter.
+                Ceci n'est <strong>PAS</strong> un jeu incrémental (incremental game) et inactif (idle game) ! Construisez plus d'extracteurs et de convoyeurs pour atteindre plus vite votre votre but.<br><br>Astuce: Gardez <strong>SHIFT</strong> enfoncé pour placer plusieurs extracteurs, et utilisez <strong>R</strong> pour les faire pivoter.
```
Comment (tobspr) [inline translations/base-fr.yaml:395]: Why is this in braces?

**Comment by PascalGrosse** [comment]
Actually, I'm not sure "jeu inactif" would be correctly understood in
french as a translation of idle game. That's why I kept the original names
in braces. I'll investigate this point further.

**Comment by PascalGrosse** [comment]
On the wikipedia page for this topic (https://fr.wikipedia.org/wiki/Jeu_incr%C3%A9mental), incremental game seems to be translated, but idle game isn't (jeu inactif is only a litteral translation). Well.... maybe it's not that important, I'll remove the braces.

**Comment by tobspr** [comment]
Alright, thank you!

**Pull request merged**